website/docs/guides/1.1.0-upgrade-guide.html.markdown

```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
 
 # MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
 
-The Terraform MongoDB Atlas Provider version 1.0.0 has a number of new and exciting features and changes.
+The Terraform MongoDB Atlas Provider version 1.1.0 has a number of new and exciting features and changes.
 
-* Cloud Backup Snapshot replaces the now deprecated Cloud Provider Snapshot.
-* Cloud Backup Snapshot Restore Job replaces the now deprecated Cloud Provider Snapshot Restore Job.
+New Features:
 
-1.1.0 also includes general improvements and bug fixes. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more specific information.
-
-## Migration from Cloud Provider Snapshot to Cloud Backup Snapshot
+* Multi-cloud deployments can now be managed with [`mongodbatlas_advanced_clusters`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_clusters)
+* Invitations can now be managed with
+[`mongodbatlas_org_invitation`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/org_invitation) and [`mongodbatlas_project_invitation`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project_invitation)
+* Projects created with [`mongodbatlas_project`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project)` can now have a default owner set with `project_owner_id`
+* [`mongodbatlas_privatelink_endpoint`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint) and [`mongodbatlas_privatelink_endpoint_service`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service) now support Google Cloud'd Private Service Connect
+* Projects created with [`mongodbatlas_project`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project)` can now have default alert creation disabled by setting `with_default_alerts_settings` to false.
+* [`mongodbatlas_search_index`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/search_index) now supports synonyms.
+* Atlas Serverless deployments, now in preview, can now be created with [`mongodbatlas_serverless_instance`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/serverless_instance)
 
-There are two possible options to upgrade from v1.0.2 to v1.1.0 from the existing resources `mongodbatlas_cloud_provider_snapshot_backup_policy` to the new ones
+Changes:
+In addition, there are changes that will require modifications to your Terraform configuration between earlier versions of the provider and this one.
```

Comment:
The bulk of these changes are covered in the next section. I think it's appropriate to link there.

consider something like:

Version 1.10 removes|deprecates the following resources:

- [`mongodbatlas_cloud_provider_snapshot`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot)
- [`mongodbatlas_cloud_provider_snapshot_restore_job`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot_restore_job)

If your Terraform configuration uses these resources, see [Migration from Cloud Provider Snapshot to Cloud Backup Snapshot](#migration-from-cloud-provider-snapshot-to-cloud-backup-snapshot) to migrate to the new resources **before you upgrade**.

Reply:
Great suggestion!  I just added a clarification that they will be removed in a future major version.